JetStreamCompose/jetstream/build.gradle

```diff
@@ -80,20 +90,24 @@ kapt {
 dependencies {
     def tv_compose_version = "1.0.0-SNAPSHOT"
 //    def tv_compose_version = "1.0.0-alpha06"
-    def core_ktx_version = "1.12.0-alpha04"
+    def core_ktx_version = "1.10.1"
     def lifecycle_runtime_ktx_version = "2.5.1"
     def compose_activity_version = "1.7.2"
     def splash_screen_version = "1.0.1"
     def nav_version = "2.5.3"
     def gson_version = "2.9.1"
     def media3_version = "1.1.0-alpha01"
+    def compose_ui_version = "1.6.0-SNAPSHOT"
+    def profileinstaller_version = "1.3.1"
 
     implementation("androidx.core:core-ktx:$core_ktx_version")
     implementation("androidx.lifecycle:lifecycle-runtime-ktx:$lifecycle_runtime_ktx_version")
     implementation("androidx.activity:activity-compose:$compose_activity_version")
-    implementation("androidx.compose.ui:ui:$compose_version")
     implementation("androidx.compose.ui:ui-tooling-preview:$compose_version")
 
+    // Compose UI (Using snapshot build for focus restoring APIs)
+    implementation("androidx.compose.ui:ui:$compose_ui_version")
```

Comment:
Since `ui` and `ui-tooling-preview` are linked together, should we use the snapshot version for the preview as well to avoid any incompatibilities?

Reply:
Done